src/main/java/org/gridsuite/securityanalysis/server/dto/LimitViolationDTO.java

```diff
@@ -34,4 +39,16 @@ public static LimitViolationDTO toDto(AbstractLimitViolationEntity limitViolatio
             .loading(limitViolation.getLoading())
             .build();
     }
+
+    public List<String> toCsvRow(Map<String, String> translations) {
+        List<String> csvRow = new ArrayList<>();
```

Comment:
Optimize array memory pre-allocation
```suggestion
        List<String> csvRow = new ArrayList<>(7);
```